user-guide/Advanced_Modules/Automation_module/FAQ/How_do_I_abort_a_running_Automation_script.md

```diff
@@ -6,18 +6,17 @@ uid: How_do_I_abort_a_running_Automation_script
 
 ## [From DataMiner 10.2.12/10.3.0 onwards](#tab/tabid-1)
 
-When a dialog box has the focus, you can abort a running Automation script in either of the following ways:
+To abort a running Automation script, close the dialog by clicking the *X* in the top-right corner.
 
-- Close the dialog box by clicking the *X* in the top-right corner.
-
-- Press ALT+F4.
-
-> [!IMPORTANT]
-> When an interactive Automation script was launched from a web app, you will have to press ESC instead of ALT+F4 to close a dialog box and abort the script. Pressing ALT+F4 would close the browser, not just the dialog box.
+> [!NOTE]
+> You can also use the following keyboard shortcuts:
+>
+> - In a web app, press ESC to close the dialog and abort the script.
+> - In DataMiner Cube, press ALT+F4 to close the dialog.
```

Comment:
I would also state here " to close the dialog and abort the script" as well.
(Same as in sentence above.)

OR

you can remove the "and abort a script" on the web app sentence, and add it on the top:
"You can also abort the script by using the following keyboard shortcuts:".
This would even be better, in my opinion.